cmake/XRootDFindLibs.cmake

```diff
@@ -39,11 +39,12 @@ find_package( CURL )
 
 if( ENABLE_CRYPTO )
   if( FORCE_ENABLED )
-    find_package( OpenSSL REQUIRED )
+    find_package( OpenSSL 1.0.2 REQUIRED )
   else()
-    find_package( OpenSSL )
+    find_package( OpenSSL 1.0.2 )
   endif()
   if( OPENSSL_FOUND )
+    add_definitions( -DHAVE_DH_PADDED )
```

Comment:
This is where I added back the definitions that was set in the OpenSSL module we had.

Reply:
Could please double check if this works well with statically linked openssl, this is something @gganis added in fea67e7844246e18ecdaa1505c9c57040cf1f29f

Reply:
Sure, I will test against a statically-linked openssl.

Reply:
I tested against a statically linked OpenSSL locally on my machine, and also on a server where I built ROOT using the builtin OpenSSL and builtin XRootD (which also uses a statically linked OpenSSL). Everything still works fine.